proto/vm/vm.proto

```diff
@@ -177,7 +170,6 @@ message ParseBlockRequest {
 message ParseBlockResponse {
   bytes id = 1;
   bytes parent_id = 2;
-  Status status = 3;
```

Comment:
should we add `reserved 3;` here?

Reply:
We don't typically do this for the VM interface, because this is gated by the rpcchainvm proto version (which will need to be increased in this release).